docs/helm/docker-desktop-deployment.md

```diff
@@ -127,68 +127,38 @@ and therefore requires a large amount of resources out-of-the-box. To reduce the
 size of the deployment so it can run on a single machine we'll need to reduce
 the number of pods deployed and the memory requirements for several others.
 
-Fortunately this can all be achieved with one, albeit large, command as shown below:
+To install the Enterprise on localhost we need to use the local-dev-values.yaml
+
+```bash
+curl -f0O https://raw.githubusercontent.com/Alfresco/acs-deployment/master/docs/helm/values/local-dev-values.yaml
```

Comment:
why -0 (zero)?

> -0, --http1.0
>       (HTTP) Tells curl to use HTTP version 1.0 instead of using its internally preferred HTTP version.

Locally I don't have any issue downloading the file from this branch with just `-fO`

Reply:
sorry, I think I mistyped on the previous review comment :D

just `-fO` is fine for this curl usage